manager/index.html

```diff
@@ -121,7 +121,7 @@
     <script src="bower_components/angular-cookies/angular-cookies.js"></script>
     <script src="bower_components/angular-sanitize/angular-sanitize.js"></script>
     <script src="bower_components/angular-route/angular-route.js"></script>
-    <script src="bower_components/datatables/media/js/jquery.dataTables.js"></script>
+    <script src="bower_components/datatables.net/js/jquery.dataTables.js"></script>
```

Comment:
need some libraries updates ?
if yes, bower should specify needed versions, what should be upgraded?